Sources/FBLPromises/FBLPromise+Wrap.m

```diff
@@ -133,7 +133,7 @@ + (instancetype)onQueue:(dispatch_queue_t)queue
                      if (error) {
                        reject(error);
                      } else {
-                       fulfill(@[ value1, value2 ]);
+                       fulfill(@[ value1 ?:[NSNull null], value2 ?: [NSNull null] ]);
```

Comment:
Could you add a single whitespace after the operator in the first array element for consistency?

```
value1 ?: [NSNull null]
```

Thanks!

Reply:
Done:)